instrumentation/ruby_kafka/lib/opentelemetry/instrumentation/ruby_kafka/patches/client.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module RubyKafka
+      module Patches
+        # The Client module contains the instrumentation patch the Producer#deliver_message method
+        module Client
+          def deliver_message(value, key: nil, headers: {}, topic:, partition: nil, partition_key: nil, retries: 1)
+            attributes = {
+              'messaging.system' => 'kafka',
+              'messaging.destination' => topic
```

Comment:
```suggestion
              'messaging.destination' => topic,
              'messaging.destination_kind' => 'topic'
```

It is also strongly recommended to add networking attributes.

Reply:
>It is also strongly recommended to add networking attributes.
Regarding networking attributes, I have access to the brokers from here.
```rb
[16] pry(#<Kafka::Client>)> brokers
=> [#<Kafka::BrokerInfo:0x00007faf231fa408 @host="kafka", @node_id=1, @port=9092>]
```

Reply:
Maybe ask @d1egoaz ?

Reply:
it's hard to tell here what specific broker will receive the message, as messages for topics go to different partitions  depending if there is a key or no (random), also, it depends on the client acks definition, as per configured in kafka, `acks=all` means that a message is sent to the broker leader for the partition, and then it waits until it's in sync with two followers, the broker leader can also change.

That's why we decided to use this parameter, and filling it with the current target kafka cluster.

https://github.com/Shopify/shopify-tracing/blob/master/lib/shopify/tracing/adapters/kafka_client_ruby_adapter.rb#L13`kk